src/Nethermind/Nethermind.Db/DbExtensions.cs

```diff
@@ -119,6 +132,13 @@ public static void Delete(this IDb db, Keccak key)
             db.Remove(key.Bytes);
         }
 
+        public static void Delete(this IDb db, long blockNumber, Keccak hash)
```

Comment:
Can get away with a skiplocalsinit on this as is fully written in GetBlockNumPrefixedKey